test_projects/example/bundlex.exs

```diff
@@ -3,17 +3,30 @@ defmodule Example.BundlexProject do
 
   def project do
     [
-      natives: natives(),
+      natives: natives()
     ]
   end
 
+
+  defp get_ffmpeg_path() do
+    case Bundlex.get_target() do
+      {_architecture, _vendor, "linux"} -> "https://github.com/BtbN/FFmpeg-Builds/releases/download/latest/ffmpeg-n4.4-latest-linux64-gpl-shared-4.4.tar.xz/"
+      _other -> :unavailable
+    end
+  end
```

Comment:
As far as I can remember, we have a precompiled ffmpeg dependency for macos

Reply:
done